utils.js

```diff
@@ -0,0 +1,246 @@
+const aws = require('aws-sdk')
+const { equals, head, includes, isNil, map, merge, not, reduce, toPairs } = require('ramda')
+const { utils } = require('@serverless/core')
+const { schema } = require('./schema')
+
+/**
+ * Get AWS clients
+ * @param {object} credentials
+ * @param {string} region
+ * @returns {object} AWS clients
+ */
+const getClients = (credentials, region = 'us-east-1') => {
+  const cloudformation = new aws.CloudFormation({ credentials, region })
+  const s3 = new aws.S3({ credentials, region })
+  return {
+    cloudformation,
+    s3
+  }
+}
+
+/**
+ * Waits CloudFormation stack to reach certain event
+ * @param {object} cloudformation
+ * @param {string} events events to wait for
+ * @param {object} config
+ * @returns {array} stack outputs
+ */
```

Comment:
Love the comments! Thank you for your clean code and your attention to details! ❤️